virttest/qemu_vm.py

```diff
@@ -661,6 +661,9 @@ def add_nic(devices, vlan, model=None, mac=None, device_id=None,
                     dev.set_param('mq', mq)
                 if vectors:
                     dev.set_param('vectors', vectors)
+                if failover:
+                    failover = 'on'
+                    dev.set_param('failover', failover)
```

Comment:
```suggestion
                    dev.set_param('failover', 'on')
```

Reply:
I have updated.

Reply:
If update like this, there will be alerts as  "1 for Unused local variable"

Reply:
I have updated, could you help revew?